template/composer.json

```diff
@@ -28,6 +27,7 @@
     "behat/mink-goutte-driver":     "*",
     "behat/mink-selenium2-driver":  "*",
     "behat/mink-browserkit-driver": "*",
+    "drush/drush":                  "dev-master",
```

Comment:
I think Moshe has said repeatedly that applications should provide their own version of Drush... it seems like we're going back on that here. If so, we should discuss whether that's still best practice.